osp/models/question.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+from osp.models.abstract_timestamp import AbstractTimestamp
+
+SA = 'char'
```

Comment:
Create a separate directory for utilities, This new dir should contain all the abstract or generic models, functions, choices, views and manager.